### PR TITLE
#[187300100] [Seller auth & Two Factor Authentication for Sellers]

### PR DESCRIPTION
**Pull request by Nadinefiona**
#### What does this PR do?

Implement Seller Auth & Two-Factor Authentication for Sellers

#### Description of Task to be completed?

This pull request introduces a two-factor authentication (2FA) feature specifically for sellers on our platform.

#### How should this be manually tested?

npm run dev 
and then do login and make sure that you are seller just to verify OTP.

#### What are the relevant pivotal tracker/Trello stories?

#187300100

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_API_URL=https://alpha-ec-be-8i4x.onrender.com
```
Comment (fab-ryan) [inline .env]: remove this file

File: src/app/(Authentication)/sellerAuth/page.tsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    }
+
+    const payload = {
+      otp: formData.otp,
+      token: token
+    };
+
+    const result = await dispatch(verifyOtp(payload));
+
+    if (verifyOtp.fulfilled.match(result)) {
```
Comment (fab-ryan) [inline src/app/(Authentication)/sellerAuth/page.tsx]: make this a state in redux

File: src/redux/slices/otpSlice.ts [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        state.error = null;
+        state.success = false;
+        state.token = null;
+      })
+      .addCase(verifyOtp.fulfilled, (state, action) => {
+        state.loading = false;
+        state.error = null;
+        state.success = true;
+        state.token = action.payload.data;
+      })
```
Comment (fab-ryan) [inline src/redux/slices/otpSlice.ts:71]: use this states in tsx file

**Review by fab-ryan — CHANGES_REQUESTED** [review]
Work on those changes

**Comment by Nadinefiona** [comment]
done

**Review by fab-ryan — APPROVED** [review]

**Review by Ruthuwamahoro — APPROVED** [review]

**Pull request merged**